### PR TITLE
[Range] Adding samples to common Range methods

### PR DESCRIPTION
**Pull request by AlexJerabek**
These samples target commonly recorded actions that didn't already have sample code.

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.rangefill.yml
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.rangefill.yml:56]:
```suggestion
            // Get the used range of the current worksheet.
```

**Review by alison-mk — APPROVED** [review]
Approving with two typo fixes.

File: docs/sample-scripts/excel-scripts.yaml
Comment (alison-mk) [inline docs/sample-scripts/excel-scripts.yaml:1375]: @AlexJerabek Those missing periods are repeated in excel-scripts.yaml and snippets.yaml. Sorry I didn't mention this earlier!

Comment (AlexJerabek) [reply docs/sample-scripts/excel-scripts.yaml:1375]: I'll fix that in an upcoming PR before this gets merged to live.

**Pull request merged**
